frontend/packages/e2e/playwright.config.ts

```diff
@@ -4,17 +4,24 @@ export default defineConfig({
   globalSetup: require.resolve('./global-setup'),
   testDir: 'tests',
   fullyParallel: true,
-  forbidOnly: !!process.env.CI,
-  retries: process.env.CI ? 2 : 0,
-  workers: process.env.CI ? 1 : undefined,
+  forbidOnly: !!process.env['CI'],
+  retries: process.env['CI'] ? 2 : 0,
+  workers: process.env['CI'] ? 1 : '50%',
```

Comment:
It seems that undefined is not accepted as a type, resulting in a compile error.
When undefined, it becomes the default value, and since the default is 50% of the CPU, the behavior does not change.

ref: https://playwright.dev/docs/api/class-testconfig#test-config-workers